test/fixtures/visualize/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "visualize",
+  "scripts": {
+    "build": "microbundle --visualize",
+    "postbuild": "mv stats.html dist/"
```

Comment:
I wonder if we can remove the need for a `postbuild` task. The docs of `rollup-plugin-visualize` don't mention it, but you can pass a directory with the `filename` setting.

https://github.com/btd/rollup-plugin-visualizer/blob/41aeee4741ab424c8e4e1ab48930ca5cbd2d8fb1/plugin/index.js#L129

Reply:
Sounds like a good plan.

@marvinhagemeister Do you have any idea about the other (unrelated) snapshots that are failing?
Last time I checked master is also failing too... so not sure what to do next.